README.md

```diff
@@ -141,7 +141,13 @@ The resemble.compare API provides a convenience function that is used as follows
 const compare = require("resemblejs").compare;
 
 function getDiff() {
-    const options = {};
+    const options = {
+      // stop comparing once determined to be > 5% non-matching; this will
+      // also enable compare-only mode and no output image will be rendered;
+      // the combination of these results in a significant speed-up in batch processing
+      misMatchThreshold: 5
```

Comment:
I think it would be clearer if there are two options, `comparisonOnly` (full comparison by default, without diff image) and `returnEarlyThreshold` (return partial diff image by default when condition is met). Both options could work without the other. Although in this case you need both there could be other use-cases requiring a little more flexibility.